loopdb/store.go

```diff
@@ -357,9 +382,22 @@ func (s *boltSwapStore) FetchLoopInSwaps() ([]*LoopIn, error) {
 				return err
 			}
 
+			// Try to unmarshal the protocol version for the swap.
+			// If the protocol version is not stored (which is
+			// the case for old clients), we'll return the
+			// passed defaultProtocolVersion instead.
+			protocolVersion, err := UnmarshalProtocolVersion(
```

Comment:
Do we need to pass in the default version? Could be more readable to just set the default first, then only call the method if anything is found in the bucket.

Reply:
The reason this is passed is to avoid repeating the same checking and defaulting code where we unmarshal the protocol version. Can slice up if you wish but imo it's cleaner to keep together. WDYT?

Reply:
I mean
```go
protocolVersion := default
versionBytes := swapBucket.Get(protocolVersionKey)
if versionBytes != nil {
   protocolVersion, err = UnmarshalProtocolVersion(versionBytes)
   if err != nil {
      return err
   }
}
```
the checks on the passed version bytes could still be performed.

Reply:
done